maps/~mapsystem/maps_antagonism.dm

```diff
@@ -22,13 +22,13 @@
 	)
 
 	var/list/potential_special_theft_targets = list(
-		"nuclear gun"             = /obj/item/gun/energy/gun/nuclear,
-		"diamond drill"           = /obj/item/pickaxe/diamonddrill,
-		"bag of holding"          = /obj/item/storage/backpack/holding,
-		"hyper-capacity cell"     = /obj/item/cell/hyper,
-		"10 diamonds"             = /obj/item/stack/material/diamond,
-		"50 gold bars"            = /obj/item/stack/material/gold,
-		"25 refined uranium bars" = /obj/item/stack/material/uranium,
+		"nuclear gun"         = /obj/item/gun/energy/gun/nuclear,
+		"diamond drill"       = /obj/item/pickaxe/diamonddrill,
+		"bag of holding"      = /obj/item/storage/backpack/holding,
+		"hyper-capacity cell" = /obj/item/cell/hyper,
+		"10 diamonds"         = list(/obj/item/stack/material/*gemstone*/, 10, /decl/material/solid/gemstone/diamond),
```

Comment:
wtf with *s? in here?

Reply:
Inline comment because the paths don't exist yet. Makes my life easier when updating the sheets PR.